src/types/reduxActions.js

```diff
@@ -182,3 +183,4 @@ export type Action =
   | { type: 'FIO/SET_LAST_EXPIRED_CHECKS', data: { [fioName: string]: Date } }
   | { type: 'FIO/CHECKING_EXPIRED', data: boolean }
   | { type: 'FIO/WALLETS_CHECKED_FOR_EXPIRED', data: { [walletId: string]: boolean } }
+  | ActionQueueAction
```

Comment:
We have been trying to gather all action definitions into this one file, rather than spreading them around throughout the program. However, I think your work is clean & focused enough that we can leave the actions where you have them. So, no need to change anything here.

Reply:
I had originally placed the files in the `plugins/` directory because I felt like this codebase was sort of like a stand-alone package, and because of this, I felt like all or most of the codebase needed to be colocated in the directory. But after moving the code to a new `controllers`, I don't feel like the package should be as "stand-alone-ish" as I originally felt like it needed to be. I'll leave it for now, but I'm also weary of my decision to spread out the redux API like I have done.

Reply:
```
// Self-Contained Package Actions:
```